beamer/health/health-check-mainnet.conf

```diff
@@ -0,0 +1,34 @@
+[default]
+agent-address=
+deployment-dir="../deployments/mainnet"
+cache-file-path="./"
+notification-system="telegram"
+
+[notification.rocketchat]
+url=""
+channel=""
+
+[notification.telegram]
+token=""
+chat_id=""
+
+[chains.arbitrum]
+rpc-url=
+explorer=https://arbiscan.io/tx/
+chain-id=42161
+start-block=46158611
+
+[chains.optimism]
+rpc-url=
+explorer=https://optimistic.etherscan.io/tx/
+chain-id=10
+start-block=49645351
+
```

Comment:
Why do we need `start-block` in the config file?

Reply:
because I'm too lazy to fetch the start block from the chain. :(

Reply:
yes, yesterday while playing spike-ball I also was struck by enlightenment and realized that I could take all this form the deployment info and don't need an RPC call. Thanks. Will look into this.